spring-cloud-aws-starters/spring-cloud-aws-starter-ecs/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there a need for starter considering it does not bring any extra dependencies?